backend/dataset/models.py

```diff
@@ -275,6 +276,62 @@ def __str__(self):
         return str(self.id)
 
 
+class Conversation(DatasetBase):
+    """
+    Dataset for storing Conversation data
+    """
+
+    domain = models.CharField(
+        verbose_name="domain",
+        max_length=1024,
+        help_text=("Domain of the conversation translation"),
+        null=True,
+        blank=True,
+    )
+    topic = models.TextField(
+        verbose_name="topic",
+        null=True,
+        blank=True,
+        help_text=("Topic of the conversation"),
+    )
+    scenario = models.TextField(
+        verbose_name="scenario",
+        null=True,
+        blank=True,
+        help_text=("Scenario of the conversation"),
+    )
+    prompt = models.TextField(
+        verbose_name="prompt",
+        null=True,
+        blank=True,
+        help_text=("Prompt of the conversation"),
+    )
+    speaker_count = models.IntegerField(
+        verbose_name="speaker_count",
+        null=True,
+        blank=True,
```

Comment:
We should not allow null values here.

Reply:
Should we also disallow null values in in `speakers_json` as well?